code/game/objects/items/weapons/implants/implant_gorilla_rampage.dm

```diff
@@ -6,14 +6,36 @@
 	uses = 1
 	implant_data = /datum/implant_fluff/gorilla_rampage
 	implant_state = "implant-syndicate"
+	COOLDOWN_DECLARE(gorilla_transform_cooldown)
 
 /obj/item/implant/gorilla_rampage/activate()
-	if(!iscarbon(imp_in))
+	if(!COOLDOWN_FINISHED(src, gorilla_transform_cooldown))
+		to_chat(usr, "<span class='notice'>[src] is still on cooldown! [(gorilla_transform_cooldown - world.time) / 10] seconds left!</span>")
 		return
+	if(ishuman(usr))
+		var/mob/living/simple_animal/hostile/gorilla/rampaging/rampaging_gorilla = new (get_turf(src))
 
-	var/mob/living/carbon/target = imp_in
-	target.visible_message("<span class='userdanger'>[target] swells and their hair grows rapidly. Uh oh!.</span>","<span class='userdanger'>You feel your muscles swell and your hair grow as you return to monke.</span>", "<span class='userdanger'>You hear angry gorilla noises.</span>")
-	target.gorillize(TRUE)
+		playsound(rampaging_gorilla, 'sound/creatures/gorilla.ogg', 50)
+		var/mob/living/carbon/human/implante = imp_in
+
+		implante.visible_message("<span class='userdanger'>[implante] swells and their hair grows rapidly. Uh oh!.</span>","<span class='userdanger'>You feel your muscles swell and your hair grow as you return to monke.</span>", "<span class='userdanger'>You hear angry gorilla noises.</span>")
+		implante.mind.transfer_to(rampaging_gorilla)
+		implante.forceMove(rampaging_gorilla)
```

Comment:
doesnt this mean you can gib the gorilla to get a free godmode body?

Reply:
no, they'll get sent directly to hell because of how simplemob gibbing works
although that is a good idea